File: java/core/src/main/java/com/github/fernthedev/core/packets/latency/PongPacket.java
package com.github.fernthedev.core.packets.latency;

import com.github.fernthedev.core.packets.Packet;
import com.github.fernthedev.core.packets.PacketInfo;

@PacketInfo(name = "PONG_PACKET")
public class PongPacket extends Packet implements LatencyPacket{

}

File: java/core/src/main/java/com/github/fernthedev/lightchat/core/packets/latency/PongPacket.kt
package com.github.fernthedev.lightchat.core.packets.latency

import com.github.fernthedev.lightchat.core.packets.PacketJSON
import com.github.fernthedev.lightchat.core.packets.PacketInfo

@PacketInfo(name = "PONG_PACKET")
class PongPacket : PacketJSON(), LatencyPacket